quiz.py

```diff
@@ -0,0 +1,148 @@
+import random
+from abc import ABCMeta, abstractmethod
+from sqlalchemy.sql.expression import exists
+from sqlalchemy.sql.functions import func
+from models.painter import Painter
+from models.painting import Painting
+from models.saint import Saint
+
+
+class MultipleChoiceQuestion:
+    def __init__(self, element, correct_option, option_list):
+        self._element = element
+        self.element = element.values()[0]
+        self.correct_option = correct_option
+        self.option_list = option_list
+
+
+class Quiz:
+    __metaclass__ = ABCMeta
+
+    def __init__(self):
+        self.number_of_rounds = 0
+        self.score = []
+
+        # current
+        self.current_question = None
+
+        self.right_guesses = sum(self.score)
+        self.wrong_guesses = len(self.score) - self.right_guesses
+
+    def get_score(self):
+        return self.score
+
+    @abstractmethod
+    def generate_question(self):
+        return None
```

Comment:
Also, you, don't really update right/wrong_guesses. This is dynamic info, make a property out of it (@property) instead, no need for calculation in the constructor